tests/e2e/stablehlo_ops/CMakeLists.txt

```diff
@@ -507,7 +507,7 @@ iree_check_single_backend_test_suite(
     "reshape.mlir"
     "reverse.mlir"
     "rng_normal.mlir"
-    "rng_uniform.mlir"
+    # "rng_uniform.mlir"  #TODO(#12509): WebGPU SPIR-V broken
```

Comment:
Error logs seemed similar to https://github.com/openxla/iree/issues/11571 (which was fixed for other op lowerings in https://github.com/openxla/iree/pull/13274)

https://github.com/openxla/iree/actions/runs/6039728876/job/16389078665?pr=14798#step:4:8384
```
Tint reported 1 error(s) for a SPIR-V program, see diagnostics:
error: extended arithmetic is not finalized for WGSL: https://github.com/gpuweb/gpuweb/issues/1565: %62 = OpIAddCarry %61 %58 %46
/work/tests/e2e/stablehlo_ops/rng_uniform.mlir:6:12: error: failed to compile SPIR-V to WGSL. Consider inspecting the shader program using -iree-hal-dump-executable-intermediates.
    %res = "stablehlo.rng"(%min, %max, %shape) {rng_distribution = #stablehlo<rng_distribution UNIFORM>} : (tensor<f32>, tensor<f32>, tensor<1xi32>) -> tensor<10xf32>
           ^
```

(totally fine to disable the webgpu test until it shows up in a workload that is prioritized, just leaving a bit more of a paper trail)

Reply:
Mmm, let me check, perhaps I didn't update git rev correctly.

Reply:
Ah ok that was for mul and this is add. @kuhar for visibility just in case I misread/this is expected to work.

Reply:
Yeah we should also add an expansion for this. Right now we only expand extended multiplication. The good news is that addition is much simpler and can make a good starter task if anyone's interested :slightly_smiling_face:

Reply:
I filed a tracking issue: https://github.com/llvm/llvm-project/issues/65154